games-strategy/crimson/crimson-0.5.3.recipe

```diff
@@ -0,0 +1,66 @@
+SUMMARY="A turn-based tactical war game"
+DESCRIPTION="The outcome of the war lies in your hands. You decide which \
+units are sent to the front lines, and when to unleash the reserves. Your \
+mission objectives range from defending strategically vital locations to \
+simply destroying all enemy forces in the area. Protect supply convoys or \
+raid enemy facilities to uncover technological secrets or fill your storage \
+bays so you can repair damaged units or build new ones in your own \
+factories. Lead your troops to victory! "
+HOMEPAGE="http://crimson.seul.org/"
+COPYRIGHT="2001-2009 Jens Granseuer"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="http://crimson.seul.org/files/crimson-$portVersion.tar.gz"
+CHECKSUM_SHA256="e82aa28f729140711e2d04c426462d96d33ce8485f5d3ca3f394210e412cefd2"
+
+ARCHITECTURES="x86_gcc2 ?x86 ?x86_64"
+
+PROVIDES="
+	crimson = $portVersion
+	cmd:bi2cf = $portVersion
+	cmd:cf2bmp = $portVersion
+	cmd:cfed = $portVersion
+	cmd:comet = $portVersion
+	app:CrimsonFields = $portVersion
+	"
+REQUIRES="
+	haiku
+	lib:libsdl
+	lib:libsdl_mixer
+	lib:libsdl_net_1.2
+	lib:libsdl_ttf
+	lib:libz
+	"
+
+BUILD_REQUIRES="
+	haiku_devel
+	devel:libsdl
+	devel:libsdl_mixer
+	devel:libsdl_net_1.2
+	devel:libsdl_ttf
+	devel:libz
+	"
+BUILD_PREREQUIRES="
+	cmd:awk
+	cmd:gcc
+	cmd:make
+	"
+
+BUILD()
+{
+	./configure --prefix=$appsDir/Crimson --enable-cfed \
+		--enable-comet --enable-bi2cf --enable-cf2bmp
+	make $jobArgs
+}
+
+INSTALL()
+{
+	make install
+	mv $appsDir/Crimson/bin/crimson $appsDir/Crimson/CrimsonFields
+	mkdir -p $binDir
+	ln -s $appsDir/Crimson/bin/bi2cf $binDir/bi2cf
+	ln -s $appsDir/Crimson/bin/cf2bmp $binDir/cf2bmp
+	ln -s $appsDir/Crimson/bin/cfed $binDir/cfed
+	ln -s $appsDir/Crimson/bin/comet $binDir/comet
```

Comment:
I don't know if HaikuPorter changes the absolute symlinks into relative symlinks, but if it doesnt (or if you wan't to avoid giving "extra" work to HaikuPorter) then you can use **`ln --relative -s `** (or **`ln -r -s `**) which will handle that gracefully and faster than HaikuPorter.